### PR TITLE
Update setup clusters scripts.

### PR DESCRIPTION
**Pull request by revit13**
This PR updates hack/setup-local-single-cluster.sh and hack/setup-local-multi-cluster.sh as follows:

- it updates the ingress resource
- it changes the catalog used to katalog
- Removes NodePort type for Vault (we use ingress to Vault instead)
- Updates KUBE_VERSION

### CODE REVIEW

**Review by Mohammad-nassar10 — APPROVED** [review]
There is an error in `golint` action, you need to increase the timeout value.

**Review by Mohammad-nassar10 — DISMISSED** [review]

File: hack/setup-local-multi-cluster.sh [hunk 1]
```diff
@@ -2,27 +2,36 @@
...
+export DEPLOY_OPENMETADATA_SERVER=0
 
 make kind-setup-multi
 kubectl config use-context kind-control
 make -C third_party/razee all
 
+# setup control cluster
 kubectl config use-context kind-control
 make cluster-prepare
-make docker-minimal-it
```
Comment (roytman) [inline hack/setup-local-multi-cluster.sh:16]: if we remove docker-minimal-it from all scripts, why do we leave it in the Makefile?

File: charts/fybrik/kind-control.values.yaml [hunk 2]
```diff
@@ -18,18 +18,21 @@ cluster:
   # Set to the name of the cluster.
   name: kind-control
   # Set to cluster zone.
-  zone: localhost
```
Comment (roytman) [inline charts/fybrik/kind-control.values.yaml:21]: in general, why do we need all these yaml files used by  Razee samples and all notebook tests files in the fybrik chart?

**Review by roytman — COMMENTED** [review]
I have several questions, let's talk

File: charts/fybrik/kind-control.values.yaml (hunk 2 above)
Comment (revit13) [reply charts/fybrik/kind-control.values.yaml:21]: @roytman as you suggested, the tests related values files are now under test/charts folder

File: Makefile [hunk 3]
```diff
@@ -243,7 +243,7 @@ ifeq ($(DEPLOY_TLS_TEST_CERTS),1)
 	cd manager/testdata/notebook/read-flow-tls && ./setup-certs.sh
 endif
 ifeq ($(DEPLOY_OPENMETADATA_SERVER),1)
-	$(MAKE) -C third_party/openmetadata all
+#	$(MAKE) -C third_party/openmetadata all
```
Comment (roytman) [inline Makefile:246]: why did you comment it out ?

Comment (revit13) [reply Makefile]: Fixed. thanks for the catch!

**Review by roytman — APPROVED** [review]

**Pull request merged**